nautobot_device_lifecycle_mgmt/models.py

```diff
@@ -446,11 +446,21 @@ class DeviceSoftwareValidationResult(PrimaryModel):
         related_name="device_software_validation",
     )
     software = models.ForeignKey(
-        to="SoftwareLCM", on_delete=models.CASCADE, help_text="Device software", null=True, blank=True, related_name="+"
+        to="SoftwareLCM", on_delete=models.CASCADE, help_text="Device software", null=True, blank=True
```

Comment:
Is this a breaking change with removing the related name item?

Reply:
Actually I'm not sure that should be removed it might have been when I was messing around with it. I'm going to add it back and test.